src/components/ChangePlan/Checkout/ContactInformation/ContactInformation.test.js

```diff
@@ -118,7 +131,7 @@ describe('Checkout component', () => {
     });
 
     // Assert
-    expect(getAllByText('validation_messages.error_required_field').length).toBe(8);
+    expect(getAllByText('validation_messages.error_required_field').length).toBe(10);
```

Comment:
How about checking against zero? Because supose we change something in the error message, the this will break. 🤔 
```suggestion
    expect(getAllByText('validation_messages.error_required_field').length).not.toBe(0);
```

Reply:
Sounds like a good idea to me too, it was one of the initial errors that came up on the first run of the tests.
I'm going to change it